prow/plugins/lgtm/lgtm.go

```diff
@@ -21,25 +21,32 @@ import (
 	"regexp"
 
 	"k8s.io/test-infra/prow/github"
+	"k8s.io/test-infra/prow/plugins"
 )
 
-const PluginName = "lgtm"
+const pluginName = "lgtm"
 
 var (
 	lgtmLabel    = "lgtm"
 	lgtmRe       = regexp.MustCompile(`(?mi)^\/lgtm\r?$`)
 	lgtmCancelRe = regexp.MustCompile(`(?mi)^\/lgtm cancel\r?$`)
 )
 
-type GitHubClient interface {
+func init() {
+	plugins.RegisterIssueCommentHandler(pluginName, handleIssueComment)
+}
+
+type gitHubClient interface {
```

Comment:
It's so that I can define a minimal interface that is what I need to use for this one plugin. Should I choose to use a custom fake client for this plugin then I only need to implement those three methods.

The alternative is to have an interface in the `github` package that has every method, but then every fake needs to implement every method!